.ci/install_kubernetes.sh

```diff
@@ -28,6 +28,13 @@ if [ "$KATA_HYPERVISOR" == "firecracker" ]; then
 	die "Kubernetes will not work with $KATA_HYPERVISOR"
 fi
 
+# Install CRI-O with its proper kubernetes version
+if [ "$ID" == "fedora" ]; then
+	echo "Install CRI-O"
+	export KUBERNETES="yes" && bash -f "${cidir}/install_crio.sh"
+	export KUBERNETES="yes" && bash -f "${cidir}/configure_crio_for_kata.sh"
```

Comment:
Should this variable continue to be exported in the current script?

Reply:
@jodh-intel yes we need to exported it

Reply:
export only once?:
```
export KUBERNETES="yes"
bash -f "${cidir}/install_crio.sh"
bash -f "${cidir}/configure_crio_for_kata.sh"
```

Reply:
@chavafg changes applied